### ISSUE TITLE
Comments in JavaScript Execution Commands Break Snippet Evaluation

### ISSUE THREAD

**Issue by manmademagic** [SilentVoid13/Templater#235]
**Plugin informations (please complete the following information):**
 - OS: Windows 10
 - Templater version: 1.7.0
 - Obsidian version: 0.12.3
 - Templater settings: 
   - Template folder location: "_Obsidian/Templater"
   - Timeout: "5"

**Describe the bug**
When inserting a JS Execution Command that contains comments, Templater inserts the entire snippet, rather than the evaluated data.

Issue also occurs when creating new note from template.

This appears to have started happening in version 1.7.0

***Working:***
```
<%*
tR += tp.date.now("YYYY-MM-DDTHH:mm:ssZ")
%>
```
Evaluates to `2021-05-23T10:05:56+08:00`

***Not Working:***
```
<%*
// this is a test
tR += tp.date.now("YYYY-MM-DDTHH:mm:ssZ")
%>
```
Evaluates to:
```
<%*
// this is a test
tR += tp.date.now("YYYY-MM-DDTHH:mm:ssZ")
%>
```

***Not Working:***
```
<%*
/* this is a test */
tR += tp.date.now("YYYY-MM-DDTHH:mm:ssZ")
%>
```
Evaluates to:
```
<%*
/* this is a test */
tR += tp.date.now("YYYY-MM-DDTHH:mm:ssZ")
%>
```

**Expected behavior**
The comments should be ignored and not affect the evaluation of the JS execution command.

**Comment by Moonbase59** [comment]
Possibly related? → https://github.com/SilentVoid13/Templater/issues/195#issuecomment-846574225

**Comment by SilentVoid13** [comment]
Hey @manmademagic, nice catch, thanks for the very nicely detailed report. This should be fixed as of Templater 1.7.1. 

Update and tell me if this works now.

**Comment by manmademagic** [comment]
> Hey @manmademagic, nice catch, thanks for the very nicely detailed report. This should be fixed as of Templater 1.7.1.
> 
> Update and tell me if this works now.

Thanks, @SilentVoid13, all working again now
